xds/internal/balancer/edsbalancer/xds_client_wrapper.go

```diff
@@ -231,8 +231,8 @@ func (c *xdsclientWrapper) handleUpdate(config *EDSConfig, attr *attributes.Attr
 	clientChanged := c.updateXDSClient(config, attr)
 
 	var (
-		restartWatchEDS   bool
-		restartLoadReport bool
+		restartWatchEndpoints bool
```

Comment:
Nit: Could we name this `restartEndpointsWatch` instead, to be consistent with everything else.

Reply:
Done